dev/README.md

```diff
@@ -37,6 +37,29 @@ container.
 
 Once ```<repo>/dev/dev.env``` has been update you can bring the stack back up using ```ocdeploy up```.
 
+Developing MongoChemClient
+--------------------------
+
+In order to visualize updates to the client before publishing, some extra steps may be needed.
+
+First, as mentioned in the section above, ```MONGOCHEMCLIENT``` in ```<repo>/dev/dev.env``` must 

```

Comment:
This step is not needed if you're going for the nginx settings route

Reply:
Okay, I got rid of that step.